src/downloadmanager.cpp

```diff
@@ -164,22 +163,55 @@ bool DownloadManager::moveMyAppPackage(QString path)
     return true;
 }
 
+bool DownloadManager::isMyApp(const QString &targetPath) const
+{
+    return QFileInfo(targetPath).completeSuffix() == QLatin1Literal("myapp");
+}
+
+bool DownloadManager::updatePlatformTransfers(const QString &targetPath) const
```

Comment:
Guess fine, but thinking if e.g. needPlatformTransfersUpdate() would be clearer. This name sounds a bit like performing an action.